common/helpers/constants.py

```diff
@@ -46,6 +46,7 @@ class FrontEndSection(Enum):
     VideoOverview = 'VideoOverview'
     AboutEventProject = 'AboutEventProject'
     CreateEventProject = 'CreateEventProject'
+    Privacy = 'Privacy'
```

Comment:
Note that constants.py needs to be kept in sync with the enums/Section.js file, so match the entries in both places.